config.js

```diff
@@ -74,8 +75,28 @@ const deprioritizeJobsFrom = configUtil.csvToArray(pe.DEPRIORITIZE_JOBS_FROM);
 // set time to live for "kue" jobs
 const JOB_QUEUE_TTL_SECONDS = pe.TTL_KUE_JOBS || DEFAULT_JOB_QUEUE_TTL_SECONDS;
 
-module.exports = {
+/*
+ * Assigns each of the different redis uses cases to a particular redis
+ * instance, if configured, or falls back to the primary redis instance.
+ */
+const redisUrls = {
+  cache: pe.REDIS_CACHE && pe[pe.REDIS_CACHE] ?
+    pe[pe.REDIS_CACHE] : (pe.REDIS_URL || DEFAULT_LOCAL_REDIS_URL),
```

Comment:
Can you give an example here, a bit hard to interpret.
So if we have pe.REDIS_URL (lets say x) set, the result will be x, else the result will be local redis url. But if pe.REDIS_CACHE is set (lets say y), do you expect the result to be y or x?